### PR TITLE
Handle no worker running for Query and Stack Trace pages

### PR DESCRIPTION
**Pull request by feedmeapples**
image

### CODE REVIEW

**Comment by rylandg** [comment]
Anyway you could make the text in the message larger?

File: client/routes/workflow/query.vue [hunk 1]
```diff
@@ -18,16 +18,30 @@
...
       </a>
     </header>
-    <pre v-if="queryResult && queryResult.payloads">{{ queryResult.payloads }}</pre>
+    <pre v-if="queryResult && queryResult.payloads">{{
+      queryResult.payloads
+    }}</pre>
     <span class="error" v-if="error">{{ error }}</span>
-    <span class="no-queries" v-if="queries && queries.length === 0">
+    <span v-if="!isWorkerRunning">
+      No worker is running for the task queue
```
Comment (cullywakelin) [inline client/routes/workflow/query.vue:26]: Consider "There is no Worker currently listening to the Task Queue" ("listening to" and "running for" maybe interchangeable)
Make sure to capitalize Worker & Task Queue

Comment (feedmeapples) [reply client/routes/workflow/query.vue]: that is a much better wording!

Comment (cullywakelin) [reply client/routes/workflow/query.vue]: My b. Probably with "are no Workers" i.e. 

"There are no Workers currently listening to the Task Queue: "

Comment (feedmeapples) [reply client/routes/workflow/query.vue]: done

**Review by cullywakelin — APPROVED** [review]

**Pull request merged**